### DISCUSSION TITLE
Dashboard Feed is still showing repos that I've unstarred

### DISCUSSION THREAD

**Discussion by anthonyivol** [community/community#106773] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Dashboard Feed is still showing repos that I've unstarred : 
<img width="720" alt="Screenshot 2024-02-08 at 15 03 46" src="https://github.com/community/community/assets/19983700/ca215e50-b3b4-48d0-b63e-a89b54b4be87">
<img width="1256" alt="Screenshot 2024-02-08 at 15 03 56" src="https://github.com/community/community/assets/19983700/50fadbd0-3ae8-4e9f-b356-3838d29fa0f9">

**Comment by humixx** [comment, marked as answer]
Star it and then un star , it helps.

**Reply by anthonyivol** [reply]
Thanks
